### PR TITLE
News Post for bnd 6.0 release

### PR DESCRIPTION
**Pull request by juergen-albert**
@bjhargrave, @pkriens and @rotty3000 I tried to pick some features for this list. Any input on missing features that should be named?

Signed-off-by: Juergen Albert <j.albert@data-in-motion.biz>

### CODE REVIEW

**Comment by bjhargrave** [comment]
> Alternative would be just to link to the Wiki, but still have a small teaser text.

I think that is better since sometimes we find and fix errors in the release notes on the wiki.

> What do you think about putting the link to the Installation page https://bndtools.org/installation.html on **every** release notes wiki page, at the top or bottom?
> 
> Yes, that would be duplicate information, but even me it took a while to figure it out. In the forum you just wrote "[the usual places](https://bnd.discourse.group/t/bnd-bndtools-6-0-0-release-now-available/160)".

https://bndtools.org/installation.html is for Bndtools only and not other artifacts like maven or gradle plugins, etc. There are lots of places in the "usual places" :-) We even have the `bnd` command in brew.

On the main README we have, https://github.com/bndtools/bnd/blob/master/README.md#release, which basically does what is necessary. It mentions the p2 update site, https://bndtools.jfrog.io/bndtools/update-latest, for Bndtools and that the maven and gradle plugins are in Maven Central. The gradle plugins are also available from the gradle plugin repo. 

Do think the release notes pages should link to https://github.com/bndtools/bnd/blob/master/README.md#release? Or do you want a more complete explanation of the "usual places" where the various release artifacts may be found?

**Comment by chrisrueger** [comment]
> Do think the release notes pages should link to https://github.com/bndtools/bnd/blob/master/README.md#release? 
Yes.

> Or do you want a more complete explanation of the "usual places" where the various release artifacts may be found?

Maybe extend the release-section on https://github.com/bndtools/bnd/blob/master/README.md#release and add a bullet point list of all the "places" you mention in your sentence:

- Bndtools Eclipse Plugin: https://bndtools.org/installation.html 
- P2 Update Site: https://bndtools.jfrog.io/bndtools/update-latest
- brew: https://formulae.brew.sh/formula/bnd
- bnd Gradle Plugin: https://plugins.gradle.org/plugin/biz.aQute.bnd or https://plugins.gradle.org/search?term=biz.aQute.bnd
- Maven Artefacts: https://mvnrepository.com/artifact/biz.aQute.bnd

Basically a single place where I can find all these clickable links :)

**Comment by bjhargrave** [comment]
Since this is very stale now, I am closing.
